### ISSUE TITLE
Travis tests fail with Twitter videos

### ISSUE THREAD

**Issue by Rudloff** [Rudloff/alltube#135]
Travis builds fail with this error:
```
1) Alltube\Test\VideoDownloadTest::testGetURL with data set #5 ('https://twitter.com/verge/sta...ideo/1', 'best', 'The_Verge_-_This_tiny_origami...056576', 'mp4', 'video.twimg.com')

Exception: ERROR: Unable to extract guest token; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

/home/travis/build/Rudloff/alltube/classes/VideoDownload.php:102

/home/travis/build/Rudloff/alltube/classes/VideoDownload.php:138

/home/travis/build/Rudloff/alltube/tests/VideoDownloadTest.php:94
```

But I can't reproduce it locally.

**Comment by Rudloff** [comment]
Here is the complete youtube-dl error:
```
[debug] System config: []

[debug] User config: []

[debug] Custom config: []

[debug] Command-line args: [u'https://twitter.com/verge/status/813055465324056576/video/1', u'-v']

[debug] Encodings: locale UTF-8, fs UTF-8, out None, pref UTF-8

[debug] youtube-dl version 2017.11.06

[debug] Git HEAD: 6a7c07c

[debug] Python version 2.7.6 - Linux-4.9.6-040906-generic-x86_64-with-Ubuntu-14.04-trusty

[debug] exe versions: ffmpeg 3.4-static, phantomjs 2.0.0, rtmpdump 2.3

[debug] Proxy map: {}

[twitter] 813055465324056576: Downloading webpage

[twitter:card] 813055465324056576: Downloading webpage

[twitter:card] 813055465324056576: Downloading m3u8 information

[twitter:card] 813055465324056576: Downloading mobile webpage

[twitter:card] 813055465324056576: Downloading main script

ERROR: Unable to extract guest token; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

Traceback (most recent call last):

  File "/home/travis/build/Rudloff/alltube/vendor/rg3/youtube-dl/youtube_dl/YoutubeDL.py", line 784, in extract_info

    ie_result = ie.extract(url)

  File "/home/travis/build/Rudloff/alltube/vendor/rg3/youtube-dl/youtube_dl/extractor/common.py", line 437, in extract

    ie_result = self._real_extract(url)

  File "/home/travis/build/Rudloff/alltube/vendor/rg3/youtube-dl/youtube_dl/extractor/twitter.py", line 225, in _real_extract

    formats.extend(self._extract_mobile_formats(username, video_id))

  File "/home/travis/build/Rudloff/alltube/vendor/rg3/youtube-dl/youtube_dl/extractor/twitter.py", line 155, in _extract_mobile_formats

    webpage, 'guest token')

  File "/home/travis/build/Rudloff/alltube/vendor/rg3/youtube-dl/youtube_dl/extractor/common.py", line 800, in _search_regex

    raise RegexNotFoundError('Unable to extract %s' % _name)

RegexNotFoundError: Unable to extract guest token; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by Rudloff** [comment]
The issue has been reported upstream: https://github.com/rg3/youtube-dl/issues/14197

**Comment by Rudloff** [comment]
Fixed in youtube-dl 2017.12.10.
